src/components/ec-main-container/ec-main-container.vue

```diff
@@ -43,23 +43,22 @@
   </div>
 </template>
 
-<script setup>
+<script setup lang="ts">
 import { useSlots } from 'vue';
 
-defineProps({
-  title: {
-    type: String,
-    default: null,
-  },
-  titleIntro: {
-    type: String,
-    default: null,
-  },
+interface MainContainerProps {
+    title?: string | null,
```

Comment:
Do we need here null as title is already optional? The same for titleIntro

Reply:
yes if we want to keep the default value as `null`, otherwise is not necessary

Reply:
This is the question: do we need null here, what is the difference between null and undefined for those properties?

Reply:
Let's remove null here.

Reply:
In this case it seems there are no differences, so yep we can remove null default values for those properties